test/src/unit-request-handlers.cc

```diff
@@ -263,4 +377,52 @@ ArraySchema HandleLoadArraySchemaRequestFx::call_handler(
       stype, resp_buf->buffer());
 }
 
+shared_ptr<ArraySchema> HandleQueryPlanRequestFx::create_schema() {
```

Comment:
is this override called by the test infrastructure? I don't see it being used in this diff.

Reply:
yes it's called by `create_array()`